infra/modules/main/lambda_deployment_iam.tf

```diff
@@ -0,0 +1,49 @@
+resource "aws_iam_user" "lambda_deployment" {
```

Comment:
I would prefer this file be called just `deployment_iam.tf` if possible, because it deploys the React frontend too, and possibly other stuff in the future as well..?

Also maybe the resource names should then be `deployment_iam` instead of `lambda_deployment`..?

Sorry for being such a nitpick. :grimacing: